quick-start-with-tidb.md

```diff
@@ -462,51 +464,43 @@ Other requirements for the target machine include:
     Input SSH password:
     ```
 
-8. Start the cluster:
+1. Start the cluster:
 
     {{< copyable "shell-regular" >}}
 
     ```shell
     tiup cluster start <cluster-name>
     ```
 
-9. Access the cluster:
-
-    - Install the MySQL client. If it is already installed, skip this step.
-
-        {{< copyable "shell-regular" >}}
-
-        ```shell
-        yum -y install mysql
-        ```
+1. Access the cluster endpoints:
 
-    - Access TiDB. The password is empty:
+    - Connect to the TiDB database using MySQL client. The password is empty:
 
         {{< copyable "shell-regular" >}}
 
         ```shell
         mysql -h 10.0.1.1 -P 4000 -u root
         ```
 
-    - Access the Grafana monitoring dashboard at <http://{grafana-ip}:3000>. The default username and password are both `admin`.
+    - Grafana: <http://{grafana-ip}:3000>. The default username and password are both `admin`.
 
-    - Access the [TiDB Dashboard](/dashboard/dashboard-intro.md) at <http://{pd-ip}:2379/dashboard>. The default username is `root`, and the password is empty.
+    - [TiDB Dashboard](/dashboard/dashboard-intro.md): <http://{pd-ip}:2379/dashboard>. The default username is `root`, and the password is empty.
 
-    - To view the currently deployed cluster list:
+1. To view the currently deployed cluster list:
```

Comment:
Please use bullet points (`-`) instead of ordered numbers (`1.`, `2.`) here because these items represent independent commands, not sequential steps that must be performed in order:

https://github.com/pingcap/docs/blob/8234ff0ee94654a55bfbdb85fad7b277a09b2bb4/quick-start-with-tidb.md?plain=1#L495-L509